application/src/main/java/org/togetherjava/tjbot/commands/moderation/modmail/ModMailCommand.java

```diff
@@ -107,68 +110,71 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
+        return true;
     }
 
-    private Optional<TextChannel> getModMailChannel(JDA jda, long userGuildI) {
-        return jda.getGuildById(userGuildI)
+    private Optional<TextChannel> getModMailChannel(JDA jda, long guildId) {
+        return jda.getGuildById(guildId)
             .getTextChannelCache()
             .stream()
             .filter(channel -> modMailChannelNamePredicate.test(channel.getName()))
             .findAny();
     }
 
-    private MessageCreateAction createMessage(SlashCommandInteractionEvent event, long userId,
+    private MessageCreateAction createModMessage(SlashCommandInteractionEvent event, long userId,
             TextChannel modMailAuditLog) {
         String userMessage = event.getOption(OPTION_MESSAGE).getAsString();
         boolean wantsToStayAnonymous = event.getOption(OPTION_STAY_ANONYMOUS).getAsBoolean();
-        String user;
-        MessageCreateAction message;
-        if (wantsToStayAnonymous) {
-            user = "Anonymous";
-            message = modMailAuditLog.sendMessageEmbeds(createModMailMessage(user, userMessage));
-        } else {
-            user = event.getUser().getAsMention();
-            message = modMailAuditLog.sendMessageEmbeds(createModMailMessage(user, userMessage))
-                .addActionRow(DiscordClientAction.General.USER.asLinkButton("Author Profile",
-                        String.valueOf(userId)));
+
+        User user = event.getUser();
+        MessageCreateAction message =
+                modMailAuditLog.sendMessageEmbeds(createModMailMessage(user, userMessage));
+
+        if (!wantsToStayAnonymous) {
+            message.addActionRow(DiscordClientAction.General.USER.asLinkButton("Author Profile",
+                    String.valueOf(userId)));
         }
         return message;
     }
 
     private void sendMessage(SlashCommandInteractionEvent event, MessageCreateAction message) {
         InteractionHook hook = event.getHook();
-        message.mapToResult()
-            .map(result -> result.isSuccess() ? "Your message has been forwarded, thanks."
-                    : "There was an issue forwarding your message, sorry. We are investigating.")
-            .flatMap(hook::editOriginal)
-            .queue(any -> {
-            }, error -> logger.warn(
-                    "There was an problem with forwarding the message to the modmail channel.",
-                    error)); // need to add on queue on success
+        message.mapToResult().map(result -> {
+            if (result.isSuccess()) {
+                return "Your message has been forwarded, thanks.";
+            }
+            logger.warn("There was an issue with forwarding users message.");
+            return "There was an issue forwarding your message, sorry. We are investigating.";
+        }).flatMap(hook::editOriginal).queue(any -> {
+        }, error -> logger.warn(
+                "There was an problem with forwarding the message to the modmail channel.", error));
```

Comment:
u dont need this, u can get rid of the extra log for that event. we get an exception stacktrace anyways.

also, the log actually only triggers on problems of `flatMap(hook::editOriginal)`, since u used `mapToResults()` on the message. so the message was already forwarded and it can only fail for editing the original message to the user. for that part we dont need extra logging.